walkthroughs/1-connecting-apps-asynchronous-messaging/walkthrough.adoc

```diff
@@ -8,7 +8,7 @@
 :amq-documentation-url: https://access.redhat.com/documentation/en-us/red_hat_amq/{amq-version}/
 
 //attributes
-:title: 1 - Connecting applications using asynchronous messaging
+:title: 1 - Connecting applications using asynchronous messaging - testing
```

Comment:
@tonyxrmdavidson Should this be removed? I don't think we want this showing up in the UI.

Reply:
👍🏽